### PR TITLE
[orchestrator] Add VPAs for orchestrator RBAC

### PR DESCRIPTION
**Pull request by JLineaweaver**
### What does this PR do?

Adds RBAC support for vertical pod autoscalers

### Motivation

Adding VPA support for the orchestrator, and we want to be able to have VPAs still show up if using the operator

### Additional Notes

Pretty simple change just a RBAC update

### Describe your test plan

Will confirm that VPAs work in staging

### CODE REVIEW

File: controllers/datadogagent/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "go.inferGopath": false
+}
```
Comment (clamoriniere) [inline controllers/datadogagent/.vscode/settings.json:3]: this file should not be committed

File: go.mod [hunk 2]
```diff
@@ -60,7 +60,6 @@ require (
 	github.com/go-logr/zapr v1.2.0 // indirect
 	github.com/go-openapi/jsonpointer v0.19.5 // indirect
 	github.com/go-openapi/jsonreference v0.19.5 // indirect
-	github.com/go-openapi/spec v0.19.3 // indirect
```
Comment (clamoriniere) [inline go.mod:63]: it doesn't seem to be need to update the go.mod for this PR.

File: controllers/datadogagent/.vscode/settings.json (hunk 1 above)
Comment (JLineaweaver) [reply controllers/datadogagent/.vscode/settings.json]: [5d5d7c2](https://github.com/DataDog/datadog-operator/pull/679/commits/5d5d7c29c89e893d42873a97a90b181296c2475a)

File: go.sum [hunk 3]
```diff
@@ -65,18 +65,10 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
 github.com/DataDog/datadog-api-client-go v1.7.0 h1:82Jkz5dFQGR+ygERKfd6JpvlVbeNLe6HaeWBypu7+2w=
 github.com/DataDog/datadog-api-client-go v1.7.0/go.mod h1:QzaQF1cDO1/BIQG1fz14VrY+6RECUGkiwzDCtVbfP5c=
-github.com/DataDog/datadog-go v4.4.0+incompatible/go.mod h1:LButxg5PwREeZtORoXG3tL4fMGNddJ+vMq1mwgfaqoQ=
```
[the remark this answers is not in the document]
Comment (JLineaweaver) [reply go.sum]: [5d5d7c2](https://github.com/DataDog/datadog-operator/pull/679/commits/5d5d7c29c89e893d42873a97a90b181296c2475a)

[the remark this answers is not in the document]
Comment (JLineaweaver) [reply go.sum]: Good catch I didn't realize this stuff got picked up in my pr

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**